helm-chart/README.md

```diff
@@ -37,8 +37,8 @@ Following are the default configuration values:
 ```yaml
 connection:
   user: postgres
-  password: ""
-  host: db.timescale.svc.cluster.local
+  password: secretPassword
```

Comment:
This should be defined as `""` in values.yaml instead.

Reply:
done